app/views/policy_areas/_form.html.erb

```diff
@@ -16,5 +16,7 @@
           class: 'select2',
           data: { placeholder: 'Choose people…' } } %>
 
+  <%= render partial: "shared/applicable_nations", locals: {f: f} %>
```

Comment:
But we should use a local var of `form` so that the partial has more context as to what `f` is.